pysat/_meta.py

```diff
@@ -768,39 +769,36 @@ def _label_setter(self, new_label, current_label, default_type,
 
     @property
     def data(self):
+        """Retrieve data.
```

Comment:
Seems weird to me to put the docstring on the property, and not the setter, since the setter typically has more variables. Checking online I see that is the standard, so ok.

Nothing to be done here, just a record of me checking on something.

Reply:
It is a little weird.  Running `help(pysat.Meta.data)` verifies that only the property is retrieved.